packages/ketcher-react/src/script/ui/state/hotkeys.ts

```diff
@@ -42,9 +42,9 @@ export function initKeydownListener(element) {
   }
 }
 
-function removeNotRenderedStruct(actionTool, event, dispatch) {
-  const { code, metaKey } = event
-  if (actionTool.tool === 'paste' && code === 'KeyS' && metaKey) {
+function removeNotRenderedStruct(actionTool, group, dispatch) {
+  const effectedTools = ['paste', 'template']
+  if (effectedTools.includes(actionTool.tool) && group?.includes('save')) {
     dispatch({
```

Comment:
If I understand correctly, `code === 'KeyS' && metaKey` is supposed to mean *the currently pressed keys are Ctrl+s*, right? I didn't know what `metaKey` is, and here is what I learned from [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey):
> On Macintosh keyboards, this is the ⌘ Command key. At least as of Firefox 48, the ⊞ Windows key is no longer considered the "Meta" key. KeyboardEvent.metaKey is false when the ⊞ Windows key is pressed.

So it seems that `metaKey` makes no sense, right? If my understanding is wrong, please let me know, thank you!

Reply:
I used external Win KB for Mac and key ⊞ works for me as original key ⌘ Command on Mac KB. May be it helps